src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -111,8 +111,14 @@ protected virtual bool MatchDescendant(ITest test)
         /// </summary>
         public static TestFilter FromXml(string xmlText)
         {
-            if (string.IsNullOrEmpty(xmlText))
-                xmlText = "<filter />";
+            const string emptyFilterXml1 = "<filter />";
```

Comment:
These two looked very identical to me, until I noticed the extra space there.  Perhaps name them emptyFilterXmlWithSpace  and .....WithoutSpace or something like that?

Reply:
changed to more descriptive names